pkg/cmd/ssh/options.go

```diff
@@ -757,17 +757,23 @@ func waitForBastion(ctx context.Context, o *SSHOptions, gardenClient client.Clie
 			return false, err
 		}
 
-		cond := corev1alpha1helper.GetCondition(bastion.Status.Conditions, operationsv1alpha1.BastionReady)
-
-		if cond == nil || cond.Status != gardencorev1alpha1.ConditionTrue {
-			lastCheckErr = errors.New("bastion does not have BastionReady=true condition")
+		switch cond := corev1alpha1helper.GetCondition(bastion.Status.Conditions, operationsv1alpha1.BastionReady); {
+		case cond == nil:
+			return false, nil
+		case cond.Status != gardencorev1alpha1.ConditionTrue:
+			lastCheckErr = errors.New(cond.Message)
+			fmt.Fprintf(o.IOStreams.ErrOut, "Still waiting: %v\n", lastCheckErr)
+			return false, nil
+		case cond.Status == gardencorev1alpha1.ConditionTrue:
```

Comment:
here it will leave the switch, right? can you add a comment? 

```go
		case cond.Status == gardencorev1alpha1.ConditionTrue:
			// leave switch
```

Reply:
https://github.com/gardener/gardenctl-v2/pull/186/commits/752aa61d74100fe4b8bd9b094c3c864dc3acaaf4